src/assets/darkmode-switch.js

```diff
@@ -57,6 +61,13 @@ let animateCssTransition = () => {
     }, 1000);
 };
 
+function isSafari() {
+    var ua = navigator.userAgent.toLowerCase();
+    if (ua.indexOf('safari') != -1 && ua.indexOf('chrome') != -1) {
+        return true;
```

Comment:
Wont this function return `undefined` if the if-condition is false?

Reply:
You're right. It works now because `undefined` is falsy. But I will add an else clause to clarify